### PR TITLE
Cartscreen hifi

### PR DESCRIPTION
**Pull request by kaeleytran**
hi this is the cart screen hifi and checkout she is very pretty but there is one functionality bug in da checkout screen but idk what it is oops

### CODE REVIEW

File: android/app/src/main/java/com/apifm/MainApplication.java [hunk 1]
```diff
@@ -7,7 +7,8 @@
 import com.oblador.vectoricons.VectorIconsPackage;
 import com.lugg.ReactNativeConfig.ReactNativeConfigPackage;
 import com.oblador.vectoricons.VectorIconsPackage;
-import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
+import com.swmansion.gesturehandler.RNGestureHandlerPackage;
+// import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
```
Comment (curesh) [inline android/app/src/main/java/com/apifm/MainApplication.java]: Uncomment this and comment out line 10

File: src/Cart/CartProduct.js [hunk 2]
```diff
@@ -23,75 +23,116 @@ const styles = StyleSheet.create({
...
     marginTop: 10,
-    marginRight: 5,
+    marginRight: 8,
+    width: '25%',
   },
   container3: {
-    //    flex: 1,
-    //    flexDirection: 'column',
+    flex: 1,
+    //  flexDirection: 'column',
```
Comment (curesh) [inline src/Cart/CartProduct.js]: Delete this and all other extraneous comments in the files

File: src/Cart/CartScreen.js [hunk 3]
```diff
@@ -112,39 +134,51 @@ export default function CartScreen({ navigation }) {
...
-      <View style={styles.subtotalContainer}>
-        <Title style={styles.subtotalText}>
-          Subtotal
-        </Title>
-        <Title style={styles.subtotal}>
-          {`$ ${parseFloat(subtotal).toFixed(2)}`}
-        </Title>
+    <View style={styles.entireScreen}>
+      <View style={styles.container}>
+        {/* <Title style={styles.titleText}>
```
Comment (curesh) [inline src/Cart/CartScreen.js]: Delete comment

**Review by curesh — DISMISSED** [review]
Hi Kaylee, this feature looks great visually! There seems to be a small bug with the price calculations, where sometimes if I change the quantity of an item by doing the following
- Delete current quantity
- Enter new quantity
There's some lag in the total price calculation, where it shows the price of the cart with the current quantity deleted, instead of the new quantity. If you can fix that bug, that'd be great!

**Review by hwang635 — APPROVED** [review]

**Pull request merged**